lib/argumentsParser.rb

```diff
@@ -0,0 +1,40 @@
+require "./lib/arguments.rb"
+
+class ArgumentsParser
```

Comment:
There's nothing wrong here that you used `class`, but may you review what a `module` is in Ruby? Can you use that here instead of `class`? What's the difference between 2 for this specific case?